ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/testHW.profile

```diff
@@ -0,0 +1,6 @@
+BootMode: Uefi
```

Comment:
We do have a `bootMode` field in SiteConfig as well - What will happen if this line and that one hold different values?

Reply:
Yes, there is a bootMode field in SiteConfig which is transformed to BMH host.Spec.BootMode. The BMH spec setting is only used in Ironic, and it does not overwrite what is configured on the host.  The bootMode field in the BIOS setting is  configured to the host.

Reply:
So should we remove this from SiteConfig if it doesn't actually affect the host's configuration?

Reply:
Yes, it is confusing to the user.